src/qililab/instruments/rohde_schwarz/sgs100a.py

```diff
@@ -78,6 +79,15 @@ def rf_on(self):
         """
         return self.settings.rf_on
 
+    @property
+    def IQ_state(self):
```

Comment:
With the same concept, this could be `modulation` or `hardware_modulation` or similar.